### ISSUE TITLE
DATA_DUMP env variable appears to have no effect

### ISSUE THREAD

**Issue by tbrowder** [tony-o/perl6-data-dump#38]
My DATA_DUMP setting:

    export DATA_DUMP="indent=4,color=false,skip-methods=true,no-postfix=true"

When using Dump in a script I still get weird characters and type names on variables.

I normally use 'note' instead of 'say' and so text goes to STDERR instead of STDOUT.

Data::Dump version 0.0.14
Raku version 2021.10
Debian Buster (10), 64-bit.

**Comment by sdondley** [comment]
Same problem here. Running macos with zsh.

**Comment by tony-o** [comment]
this should be fixed with v0.0.15

**Comment by tbrowder** [comment]
So updated, will test later...
Thanks!

**Comment by tbrowder** [comment]
Hm, when I uninstall it and install the updated one I get:

```
bigtom:xmas$ zef uninstall Data::Dump
===> Uninstalled from inst#/home/tbrowde/.raku
Data::Dump:ver<0.0.14>:auth<zef:tony-o>
bigtom:xmas$ zef install Data::Dump
===> Searching for: Data::Dump
===> Staging Data::Dump:ver<0.0.15>:auth<zef:tony-o>
===> Staging [OK] for Data::Dump:ver<0.0.15>:auth<zef:tony-o>
===> Testing: Data::Dump:ver<0.0.15>:auth<zef:tony-o>
===> Testing [FAIL]: Data::Dump:ver<0.0.15>:auth<zef:tony-o>
Aborting due to test failure: Data::Dump:ver<0.0.15>:auth<zef:tony-o> (use --force-test to override)
```

**Comment by tony-o** [comment]
weird, will take a look - it should work with --/test for the time being

**Comment by tony-o** [comment]
@tbrowder i'm not able to repro, can you do the output with ` zef install -v Data::Dump` ?

I wonder if you have that DATA_DUMP flag set and it's messing with the tests.

**Comment by tbrowder** [comment]
Yes it is set, shall I turn it off?  Seems like it would be better if you weren't affected by the user's settings, IMHO. Can't you just set the '%*ENV' hash as you need to?

**Comment by tony-o** [comment]
yea that's probably a good idea, as a work around you can install with `--/test` or just do the `DATA_DUMP='' zef install Data::Dump` until i'm able to publish an update

**Comment by tbrowder** [comment]
Ok, it installed okay with the --/test.

**Comment by tony-o** [comment]
@tbrowder thank you for the report, should be fixed now with 0.0.16

**Comment by tbrowder** [comment]
You are very welcome, Tony! And thank you for the fix. I've just upgraded to the new version on my main hosts.
